.prettierignore

```diff
@@ -0,0 +1,56 @@
+node_modules
+/coverage
+/docs
+*.md
+*.html
+*.yml
+/airtable
```

Comment:
Can we make sure to have addressing this planned? I don't want this to become something we forget about

Reply:
A few tickets around this are being written up for milestone 2 of this effort. Fortunately, getting these apps formatted should be as simple as removing them from `.prettierignore` and re-rerunning `npm run format`.